src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3328,7 +3328,8 @@ private void BuildExtendedProperties(IIndentedStringBuilder outerwriter, XamlObj
 							{
 								writer.AppendLineInvariantIndented("// Load {0}", member.Value);
 							}
-							else if (member.Member.Name == "Uid")
+							else if (member.Member.Name == "Uid"
```

Comment:
This might need to be fixed as well:

https://github.com/unoplatform/uno/blob/4dd818472c94c70dc5d4c6237480130bd41e8e6b/src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs#L5885

Reply:
It should be `var localizedObject = FindMember(objectDefinition, "Uid", XamlConstants.XamlXmlNamespace)`

A test that covers the code path I mentioned may be needed as well.

Reply:
Hi @Youssef1313 ! Thanks for the feedback. I'm so sorry for not being able to do this earlier. 

> A test that covers the code path I mentioned may be needed as well.

Do you mean to add a test for the `GenerateFile` method? Not sure how to add one for this. Can you give some tips please?

Reply:
The tests that you added is good enough, thanks!